lms/djangoapps/courseware/tabs.py

```diff
@@ -333,6 +333,8 @@ def link_func(course, reverse_func):
     @classmethod
     def is_enabled(cls, course, user=None):
         """Returns true if this tab is enabled."""
+        if not super().is_enabled(course, user=user):
+            return False
         return RELATIVE_DATES_FLAG.is_enabled(course.id)
```

Comment:
In MT fashion, thought I'd mention you could 1 line this to 
```
return super().is_enabled(course, user=user) and RELATIVE_DATES_FLAG.is_enabled(course.id)
```
But very okay with the way it is